UPGRADE.md

```diff
@@ -0,0 +1,131 @@
+# Phoenix 1.5.x to 1.6 upgrade instructions
+
+## Update your deps
+
+In `mix.exs`, update your `phoenix`, `phoenix_html`, `telemetry_metrics`, `telemetry_poller` and `phoenix_live_dashboard` deps, and add `phoenix_live_view`:
+
+```elixir
+def deps do
+    [
+      {:phoenix, "~> 1.6.0"},
+      ...
+      {:phoenix_html, "~> 3.0"},
+      {:phoenix_live_view, "~> 0.16.4"},
+      {:phoenix_live_dashboard, "~> 0.5"},
+      {:telemetry_metrics, "~> 0.6"},
+      {:telemetry_poller, "~> 0.5"},
+      ...
+    ]
+end
+```
+
+Next, run `mix deps.get` to grab your new deps.
+
+
+## Rename your `.html.eex` and `.html.leex` templates to `.html.heex` (optional)
+
+While leex templates have been deprecated, this step is optional. For the most part, existing templates should continue to work, but the HTML-aware HEEx engine will enforce valid markup and is more strict in the elixir expressions that appear within an open and closing tag.
+For example, the following code will raise:
+
+```eex
+<div id="<%= @id %>">
+```
+
+Instead of the standard `<%= %>` EEx expressions, elixir expressions inside tags can only appear withing `{}`, such as:
```

Comment:
```suggestion
Instead of the standard `<%= %>` EEx expressions, elixir expressions inside tags can only appear with `{}`, such as:
```

Reply:
Let's first move the current version, then make fixes and improvements.